docs/trulens_eval/getting_started/dashboard/index.md

```diff
@@ -0,0 +1,108 @@
+# Viewing Results
+
+TruLens provides a broad set of capabilities for evaluating and tracking applications. In addition, TruLens ships with native tools for examining traces and evaluations in the form of a complete dashboard, and components that can be added to streamlit apps.
+
+## TruLens Dashboard
+
+To view and examine application logs and feedback results, TruLens provides a built-in Streamlit dashboard. That app has two pages, the Leaderboard which displays aggregate feedback results and metadata for each application version, and the Evaluations page where you can more closely examine individual traces and feedback results. This dashboard is launched by `tru.run_dashboard()`, and will run from a database url you specify with `Tru()`.
```

Comment:
Can make `Tru` and `tru.run_dashboard` be links to the API reference: [Tru()][trulens_eval.tru.Tru] and [tru.run_dashboard][trulens_eval.tru.Tru.run_dashboard].